ceremonies/src/lib.rs

```diff
@@ -387,6 +381,36 @@ pub mod pallet {
 			Self::deposit_event(Event::RewardsIssued(cid, meetup_index, reward_count));
 			Ok(().into())
 		}
+
+		#[pallet::weight((1000, DispatchClass::Operational, Pays::No))]
+		pub fn set_inactivity_timeout(
+			origin: OriginFor<T>,
+			inactivity_timeout: InactivityTimeoutType,
+		) -> DispatchResultWithPostInfo {
+			<T as pallet::Config>::CeremonyMaster::ensure_origin(origin)?;
+			<InactivityTimeout<T>>::put(inactivity_timeout);
```

Comment:
`T as pallet`, was probably needed because we also have the `CeremonyMaster` in the scheduler pallet?

Reply:
yep